src/scripts/components/VideoForm.js

```diff
@@ -1,5 +1,3 @@
-/* @flow */
```

Comment:
Why removed `flow`?

Reply:
because we are not using this component anymore, but i've given flow error. It's just there until we merge this, because maybe there is some useful code inside